### ISSUE TITLE
Regarding the issue of datasets（关于数据集问题）

### ISSUE THREAD

**Issue by vagrancy7** [yysdck/SFFNet#5]
请问你们数据集的切割是1024*1024还是512*512？
<img width="383" alt="datasets_problem" src="https://github.com/user-attachments/assets/381b3b49-14fc-4442-9aed-ecfb92bcd18b" />

**Comment by yysdck** [comment]
> 请问您的数据集的分割是1024×1024还是512×512？（请问你们数据集的切割是1024×1024还是512×512？） <img alt="数据集问题" width="383" src="https://private-user-images.githubusercontent.com/43715578/397195044-381b3b49-14fc-4442-9aed-ecfb92bcd18b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ1ODY3NzYsIm5iZiI6MTczNDU4NjQ3NiwicGF0aCI6Ii80MzcxNTU3OC8zOTcxOTUwNDQtMzgxYjNiNDktMTRmYy00NDQyLTlhZWQtZWNmYjkyYmNkMThiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE5VDA1MzQzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc5ZjZjNDE2NGYzYWI2MmY1YTRmOGM0NzlmNzdmMmJlNDViMWExYTgxYjJhYjI0ZDA2ZjA5NGE5NjMzYjQ1Y2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Gg_xGdNzDXQhIDAFvwcwUjE1eJ-FJbRz7-lyJmQXzOA">

In the actual experiment, the dataset was initially cropped to a uniform image size of 1024×1024. Subsequently, to accommodate hardware resource limitations, a different cropping strategy was adopted during the training phase. Specifically, during training, we randomly cropped the original 1024×1024 images to 512×512 to reduce memory usage and improve training efficiency. However, during the testing phase, in order to ensure the accuracy and consistency of the evaluation results, 1024×1024 images were used for inference.

**Comment by vagrancy7** [comment]
Thank you for your previous response. Another question, may I ask if it's convenient to ask? What are the step sizes for training and testing? Is it necessary to specify that the test step size is the same as the size of the test segmentation?

感谢您之前的回答。另外还有一个问题，不知道方便问吗。请问训练和测试的步长分别都是多少？是否需要规定测试步长与测试分割的尺寸一样呢？

**Comment by yysdck** [comment]
> Thank you for your previous response. Another question, may I ask if it's convenient to ask? What are the step sizes for training and testing? Is it necessary to specify that the test step size is the same as the size of the test segmentation?
> 
> 感谢您之前的回答。另外还有一个问题，不知道方便问吗。请问训练和测试的步长分别都是多少？是否需要规定测试步长与测试分割的尺寸一样呢？

The stride for the test set should be set to the image crop size (i.e., 1024) to ensure a comprehensive evaluation of the entire image during the inference phase, thereby avoiding inference inconsistencies caused by image boundary effects. For the training set, the stride can be set to 1024, 512, or any other size. In our experiment, we used a stride of 512 for the training set.

**Comment by vagrancy7** [comment]
I am deeply grateful for your response, which has been exceedingly beneficial to me.
